app/_includes/gateway/failover-targets.md

```diff
@@ -0,0 +1,14 @@
+<!-- used on /gateway/entities/upstream/ and /gateway/entities/targets/ -->
+You can set `failover: true` to indicate that a Target should be used as a failover (backup) Target in case the other, regular targets (`failover: false`) associated with an Upstream are unhealthy. The failover Target is only used when *all** regular Targets are unhealthy. Only the [latency](/gateway/entities/upstream/#latency), [least-connections](/gateway/entities/upstream/#least-connections), and [round-robin](/gateway/entities/upstream/#round-robin) load balancing algorithms use failover Targets. 
```

Comment:
```suggestion
You can set `failover: true` to indicate that a Target should be used as a failover (backup) Target in case the other, regular targets (`failover: false`) associated with an Upstream are unhealthy. The failover Target is only used when *all** regular Targets are unhealthy. 
Failover Targets are supported by the [latency](/gateway/entities/upstream/#latency), [least-connections](/gateway/entities/upstream/#least-connections), and [round-robin](/gateway/entities/upstream/#round-robin) load balancing algorithms. 
```
Rephrased from "only these algorithms support this" to sound a bit less negative.